lib/Conversion/StablehloToTcp/StablehloToTcp.cpp

```diff
@@ -22,7 +22,7 @@
 
 namespace mlir {
 
-#define GEN_PASS_DEF_CONVERTTCPTOLINALG
+#define GEN_PASS_DEF_CONVERTSTABLEHLOTOTCP
```

Comment:
Wondering how it was working earlier.

Reply:
No idea. I'm thinking if it is even needed here.

Reply:
If I were to guess, it has to do with using a unique define to avoid including tablegen generated headers multiple times... Not sure though...